.github/workflows/ci.yml

```diff
@@ -69,6 +74,12 @@ jobs:
       
     - name: Secret detection
       uses: gitleaks/gitleaks-action@v1.6.0
+      
+    - name: Perform License Scanning
+      run: license_finder -r
+
```

Comment:
Does this need the allowlist adding? Also does it need a mention in the README how it can be added to?

Reply:
Good call! I have updated CONTRIBUTING.md with information on license/package scanning!